### ISSUE TITLE
Possible conflict between our redirect and those in .htaccess

### ISSUE THREAD

**Issue by House3272** [certbot/certbot#2210]
Where or how is this implemented?

I enabled it when configuring for the first time.
I'd like to turn it off and use .htaccess since the two seem to be conflicting.

**Comment by pde** [comment]
It'll be a RewriteRule in your http vhost.  It's implemented [here](https://github.com/letsencrypt/letsencrypt/blob/master/letsencrypt-apache/letsencrypt_apache/configurator.py#L989).  Please tell us what the conflict you're seeing is, so that we can try to prevent creating it!

**Comment by pde** [comment]
(You can also use `letsencrypt rollback` to revert the change)

**Comment by pde** [comment]
@House3272 nudge.  Can you provide more info on what the broken state you wound up in was?

**Comment by House3272** [comment]
Sorry, I don't have the exact setup anymore, but basically, on Ubuntu 14.04 I had an ec2 instance setup with a domain name.
Couldn't get both the www. and the bare domain to register simultaneously using the auto-apache, so I think I picked the bare domain. 
Then when it asked me if I wanted to do https only in the prompt, I said yes.
Later, I was trying to redirect all requests to the www. subdomain back to the bare one, which worked until I tried it from https://www.
In an attempt to correct for that, I wrote another line in .htaccess but instead got hit with a redirect loop error.
I think this was mostly my fault since I actually didn't know about apache's various ...default.conf files, thus the loop was probably due to overlapping rules.
